src/arrayMethodPop.js

```diff
@@ -5,7 +5,13 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    const lastItem = this[this.length - 1];
+
+    if (this.length !== 0) {
```

Comment:
перевірку роблять до того як працювати зі значеннями та створювати змінні
```suggestion
    if (this.length) {
```